content/spin/designing-spin-apps.md

```diff
@@ -0,0 +1,35 @@
+title = "Designing Spin Applications"
+template = "spin_main"
+date = "2023-05-05T00:01:01Z"
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/spin/designing-spin-apps.md"
```

Comment:
Double slash intentional?

Reply:
I think that's just carried over from some other file. It needs to be fixed across multiple files on the repo